pages/docs/running/orchestration.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: "ELT & ETL Orchestration"
+date: "2021-11-11"
+pullQuote: "It's easy to integrate Grouparoo into your existing ELT/ELT process"
+---
+
+It's easy to integrate Grouparoo into your existing ELT/ELT process! Grouparoo is a core part of the [modern data stack](/solutions/modern-data-stack) and provides a number of ways to integrate with the rest of your data infrastructure, and is compatible with a number of different orchestrators - tools that schedule and coordinate different parts of your data pipeline.
+
+There are 4 main ways tell Grouparoo when to check for new or updated data:
+
+- [Triggering Schedules via Time](#triggering-schedules-via-time)
+- [Triggering Schedules via the API](#triggering-schedules-via-the-api)
+- [Triggering Schedules via a Query](#triggering-schedules-via-a-query)
+- [Running Grouparoo directly via the CLI](#running-grouparoo-directly-via-the-cli)
+
+## Triggering Schedules via Time
+
+The simplest way to check for new data is create a recurring [Schedule](/docs/getting-started/product-concepts#schedule) within Grouparoo. A recurring Schedule will check your Source every-so-often for new data and import it.
+
+For example, this configuration below would check the `users` table in the data warehouse for new data every 30 minutes. We know data is new because looking at the `updated_at` column and storing the largest value seen as a [High Water Mark](/docs/getting-started/product-concepts#high-watermark)
+
+```js
+// from config/sources/users.json
+
+[
+  {
+    class: "Source",
+    id: "users",
+    modelId: "users",
+    name: "Data Warehouse Users",
+    type: "redshift-import-table",
+    appId: "data_warehouse",
+    options: {
+      table: "users",
+    },
+    mapping: {
+      id: "user_id",
+    },
+  },
+  {
+    class: "Schedule",
+    id: "users_schedule",
+    name: "Users Schedule",
+    sourceId: "users",
+    recurring: true,
+    recurringFrequency: 1800000, // 30 minutes in ms
+    confirmRecords: false,
+    options: {
+      column: "updated_at",
+    },
+    filters: [],
+  },
+];
+```
+
+This method is the simplest and is built into Grouparoo directly.
+
+## Triggering Schedules via the API
+
+Using the Grouparoo API, you can choose to run a [Schedule](/docs/getting-started/product-concepts#schedule) whenever it is right for you. A common use-case is to trigger all of your Grouparoo Schedules to look for new & updated Records at the conclusion of a [dbt](https://www.getdbt.com/) transform step via and Orchestrator like [Airflow](https://airflow.apache.org/) or [Dagster](https://dagster.io/).
+
+The `/api/v1/schedules/run` endpoint can be used to trigger a Run for your Schedules. You can run all of your Schedules or provide a list of specific `scheduleIds` to run.
+
+First, create an [API Key](/docs/getting-started/product-concepts#api-key) with the `write` permission on `Sources`, and then you can use the API as documented below:
```

Comment:
I think it would be more helpful to link to how to generate one.  Or maybe the product concept should link there?

Reply:
Agree! Added.